tools/postinstall-android.js

```diff
@@ -109,10 +109,10 @@ shelljs.echo("include \":SmartSync\"\n").toEnd(path.join(appProjectRoot, 'settin
 shelljs.echo("include \":SalesforceHybrid\"\n").toEnd(path.join(appProjectRoot, 'settings.gradle'));
 
 console.log('Moving Gradle wrapper files to application directory');
-shelljs.mv(path.join(pluginRoot, 'gradle.properties'), appProjectRoot);
-shelljs.mv(path.join(pluginRoot, 'gradlew.bat'), appProjectRoot);
-shelljs.mv(path.join(pluginRoot, 'gradlew'), appProjectRoot);
-shelljs.mv(path.join(pluginRoot, 'gradle'), appProjectRoot);
+shelljs.cp(path.join(pluginRoot, 'gradle.properties'), appProjectRoot);
+shelljs.cp(path.join(pluginRoot, 'gradlew.bat'), appProjectRoot);
+shelljs.cp(path.join(pluginRoot, 'gradlew'), appProjectRoot);
+shelljs.cp(path.join(pluginRoot, 'gradle'), appProjectRoot);
```

Comment:
You'll need to add the ```'-R'``` parameter to the copy step, since some of them are directories, like ```gradle```. For example, ```shelljs.cp('-R', path.join(pluginRoot, 'gradle'), appProjectRoot);```.